index.js

```diff
@@ -1,39 +1,44 @@
 (function(root) {
+  var docEl = document && document.documentElement;
   var scrollbarSize;
+  var scrollTop;
 
   function getScrollbarSize() {
     if (typeof scrollbarSize !== 'undefined') return scrollbarSize;
 
-    var doc = document.documentElement;
```

Comment:
These invocations of `document` were deliberately placed inside the functions in order to ensure compatibility with server-rendered React apps.